### PR TITLE
Fix const corectness

### PR DESCRIPTION
**Pull request by ZaMaZaN4iK**
Hello.

In this PR added some missing const qualifiers.

BTW, when will be removed dependency on libfolly?

### CODE REVIEW

File: OomDetector.cpp [hunk 1]
```diff
@@ -148,21 +148,21 @@ bool OomDetector::isHeuristicOOM(
...
     OomContext& octx) {
-  int threshold = tunables_->get<int>(Tunables::THRESHOLD);
-  int high_threshold = tunables_->get<int>(Tunables::HIGH_THRESHOLD);
-  int high_threshold_duration =
+  const int threshold = tunables_->get<int>(Tunables::THRESHOLD);
+  const int high_threshold = tunables_->get<int>(Tunables::HIGH_THRESHOLD);
+  const int high_threshold_duration =
       tunables_->get<int>(Tunables::HIGH_THRESHOLD_DURATION);
-  float fast_fall_ratio = tunables_->get<float>(Tunables::FAST_FALL_RATIO);
+  const float fast_fall_ratio = tunables_->get<float>(Tunables::FAST_FALL_RATIO);
```
Comment (danobi) [inline OomDetector.cpp]: Can you wrap the line? I'll put up a clang-format config later.
```
const float fast_fall_ratio = 
    tunables_->get<float>(Tunables::FAST_FALL_RATIO);
```

File: OomDetector.cpp [hunk 2]
```diff
@@ -46,11 +46,11 @@ bool OomDetector::isOOM(OomdContext& ctx) {
   auto current = Fs::readMemcurrent(cgroup_path_);
```
Comment (danobi) [inline OomDetector.cpp]: May as well make `current` and `pressure` const as well

Comment (ZaMaZaN4iK) [reply OomDetector.cpp]: Done

File: OomDetector.cpp (hunk 1 above)
Comment (ZaMaZaN4iK) [reply OomDetector.cpp]: Done

**Comment by ZaMaZaN4iK** [comment]
BTW, @facebook-github-bot provided broken link for "re-importing PR". Is it normal?

**Comment by ZaMaZaN4iK** [comment]
@danobi When you remove `XLOG` what do you plan to use for logging instead?

**Comment by danobi** [comment]
Applying the patch, thanks. It should be visible in an hour or so.

I think, the link is meant for FB employees. Not sure why our tooling decides to share those links publicly.

I intend to write my own version of XLOG. All I need is an ostream interface and off-thread flushing. Should be as simple as a cond var + a threadsafe queue.

**Comment by ZaMaZaN4iK** [comment]
For logging you can try e.g. spdlog. And try to use Conan for dependency management.

**Pull request closed without merging**